zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/value/management/CheckpointType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol.record.value.management;
+
+public enum CheckpointType {
+  NO_BACKUP,
```

Comment:
:wrench: I'd heavily prefer `MARKER` or something else that is less prescriptive about what will happen as a result of this checkpoint and focuses more on where this checkpoint is coming from.

Maybe at some point, for whatever reason, we will want to take a backup for a `MARKER` checkpoint.